### DISCUSSION TITLE
Version 2.0 released!

### DISCUSSION THREAD

**Discussion by asyncliz** [material-components/material-web#5686] [Announcements]
Hello everyone!

We just released our first [major version change: 2.0](https://github.com/material-components/material-web/releases/tag/v2.0.0)! 🥳

This version only has a couple of minor breaking changes; we expect the majority of folks to update without issue. We put together a [v2 migration guide](https://github.com/material-components/material-web/blob/main/migrations/v2/README.md) that gives more instructions on the changes and how to mitigate them.

---

I know many folks are anxiously awaiting news since our [maintenance mode announcement](https://github.com/material-components/material-web/discussions/5642). We don't have any significant updates for that yet.

In the meantime, this quarter we're working on building up our internal community of contributors (you may have already noticed a couple of new names on commits!) and setting up the infrastructure and documentation for MWC to better support contributions.

Thank you all for your ongoing patience and support!

**Comment by CarlKho-Minerva** [comment]
Can't wait! Wishing the team the best of luck. 🎉

**Comment by bakhtiaryd9-ui** [comment]
Hello, I hope you're doing well.
Unfortunately, the project has taken so long that my hair has turned white, and it's still not finished.
Please don’t leave things half-done and keep people waiting.
I'm still waiting for the final result or an update. Thank you.

**Comment by bakhtiaryd9-ui** [comment]
It seems to me that you're not able to find the core foundation of the work, and that's why you keep hitting a dead end.
